src/Components/MultiplayerComponent.h

```diff
@@ -1,5 +0,0 @@
-#pragma once
-
-struct MultiplayerComponent
```

Comment:
czemu to usunięte?

Reply:
Przestał być używany w momencie wprowadzenia SynchronisedEventów przez Mikołaja.